src/utils/wallet.ts

```diff
@@ -45,3 +48,16 @@ export const formatWalletAddress = (address: string) => {
   if (!address) return '';
   return `${address.substring(0, 6)}...${address.substring(address.length - 4)}`;
 };
+
+export const PROMOTED_WALLETS: PromotedWallet[] = [
+  {
+    name: 'MetaMask',
+    logo: metamaskLogo,
+    downloadUrl: 'https://metamask.io/download/',
+  },
+  {
+    name: 'HOT Wallet',
+    logo: hotWalletLogo,
+    downloadUrl: 'https://hot-labs.org/extension/',
+  },
+];
```

Comment:
Wallets that (if a user doesn't have installed) will be displayed with a "Get" button to link to download